pkg/virt-controller/services/virtiofs.go

```diff
@@ -27,8 +27,8 @@ func generateVirtioFSContainers(vmi *v1.VirtualMachineInstance, image string, co
 	containers := []k8sv1.Container{}
 	for _, volume := range vmi.Spec.Volumes {
 		if _, isPassthroughFSVolume := passthroughFSVolumes[volume.Name]; isPassthroughFSVolume {
-
-			container := generateContainerFromVolume(&volume, image, config)
+			resources := resourcesForVirtioFSContainer(vmi.IsCPUDedicated(), *vmi.Status.QOSClass == k8sv1.PodQOSGuaranteed, config)
```

Comment:
You most likely want to do following
`*vmi.Status.QOSClass == k8sv1.PodQOSGuaranteed` -> `vmi.IsCPUDedicated() || vmi.WantsToHaveQOSGuaranteed`

Reply:
thx for comment, i give a try

Reply:
@xpivarc you absolutely right. Big savior :)